test/test_docker_images.py

```diff
@@ -49,6 +49,6 @@ def test(tmp_path):
     expected_wheels = [
         w
         for w in utils.expected_wheels("spam", "0.1.0", manylinux_versions=["manylinux2010"])
-        if "-pp" not in w and "-cp39-" not in w
+        if "-pp" not in w and "-cp39-" not in w and "-cp310-" not in w
```

Comment:
Same comment about finding a way to not grow this. Maybe `"-cp31" not in w`? That will hold us until it doesn't matter anymore.

Reply:
Your suggestion works for me. It's not intuitive though.

```suggestion
        if "-pp" not in w and "-cp39-" not in w and "-cp31" not in w
```

Reply:
I don't really mind about the growing, but I'd like the logic to match the CIBW_SKIP above. Or, could we flip it instead?

So we'd do `"CIBW_BUILD": "cp{36,37,38}-*"`, and

```suggestion
        if "-cp36-" in w or "-cp37-" in w or "-cp38-" in w
```